### DISCUSSION TITLE
How to swap the node quickly?

### DISCUSSION THREAD

**Discussion by nothingness6** [comfyanonymous/ComfyUI#3817] [Q&A]
Every time I pull up the workflow, some nodes are unavailable, and I have to replace them with the new ones. 
The thing is, it wastes time to do that. You have to re-link the new nodes.
Is there a way to swap it at once?

**Comment by ltdrdata** [comment]
https://www.youtube.com/watch?v=SZ5dARcilpY
https://www.youtube.com/watch?v=-R4VMymHwLY

ComfyUI-Manager provides several features for that.

**Comment by ltdrdata** [comment]
> @ltdrdata I don't think you understand their question. They are asking if there is a way to swap similar nodes by some kind of holding of a hotkey to move connections to the new node AUTOMATICALLY without taking the time to move them. Efficiency is necessary.

You need to watch second video.

Additionally, that feature has been separated from ComfyUI-Manager.

https://github.com/ltdrdata/comfyui-connection-helper
